run_cookiecutter_example.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env python3
```

Comment:
Any reason not to keep this line?

Reply:
I probably could have left this but it tripped me up in the workflow by sidestepping the conda environment.

Reply:
In reality it may not have that effect at all. I'll see what happens if I put it back.

Reply:
restored!